lib/bulk-apply-applications.js

```diff
@@ -361,12 +323,10 @@ function bulkApplyApplications (args, callback) {
             };
             //create the payload for the dojo
             var dojoPayload = _.merge(payloadChanges, commonPayload);
-            //send the dojo email
-            seneca.act({role: 'kue-queue', cmd: 'enqueue', name: 'bulk-apply-applications-kue', msg: _.clone({role: 'cd-dojos', cmd: 'send_email', payload: dojoPayload}, function (err, res) {
+            //queue email for sending
+            sendQueue({cmd: 'enqueue', name: 'bulk-apply-applications-kue', msg: _.clone({role: 'cd-dojos', cmd: 'send_email', payload: dojoPayload}, function (err, res) {
                 if (err) {
-                  return err;
-                } else {
-                  return res;
+                  done(err);
```

Comment:
Every async.parallel callbacks are "wrong".
See https://caolan.github.io/async/docs.html#parallel example and use a cb for every functions of the series/parallel